target/linux/ramips/mt7621/base-files/etc/rc.button/lights_toggle

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ "${ACTION}" = "released" ] || exit 0
+
+. /lib/functions.sh
+
+case $(board_name) in
+	tplink,ec330-g5u-v1)
+		led_light="/sys/class/gpio/led-light/value"
+		if [ $(cat $led_light) = 0 ]; then
+			echo 1 > $led_light
+		else
+			echo 0 > $led_light
```

Comment:
It can be simplified to `echo $(($(cat $led_light) ^ 1)) > $led_light`.

Reply:
Thanks. I like this. Done.